stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -787,3 +826,114 @@ def get_queryables(
             "description": "Queryable names for the example STAC API Item Search filter.",
             "properties": {},
         }
+
+
+@attr.s
+class BaseAggregationClient(abc.ABC):
```

Comment:
this should be in stac_fastapi.extensions IMO

Reply:
opened https://github.com/stac-utils/stac-fastapi/issues/703 to do the same for the filter extension

Reply:
@jamesfisher-gis could you move this outside the `types` submodule?

Reply:
Yup! This afternoon I will have time to update the code following what you did with the filter extension

Reply:
Done. Thanks @vincentsarago. Let me know if you have any other comments